lib/console/organization_hotspots/organization_hotspots.ex

```diff
@@ -34,4 +34,28 @@ defmodule Console.OrganizationHotspots do
   def delete_org_hotspot(%OrganizationHotspot{} = org_hotspot) do
     Repo.delete(org_hotspot)
   end
+
+  def claim_org_hotspots(hotspot_addresses, organization) do
+    new_organization_hotspots = Enum.reduce(hotspot_addresses, [], fn hotspot_address, acc ->
+      if Repo.get_by(OrganizationHotspot, hotspot_address: hotspot_address, organization_id: organization.id) == nil do
```

Comment:
can we run 1 query to find all the OrgHotspots and reduce through the results to create the list of attrs?

Also, a hotspot may not be nil even if it's not claimed because it could have an alias

Reply:
just the first part of my comment then!